src/ring_theory/fractional_ideal.lean

```diff
@@ -529,6 +537,17 @@ begin
     apply coe_ideal_le_one },
 end
 
+variables (S P)
+
+/-- `coe_ideal_hom (S : submonoid R) P` is `coe : ideal R → fractional_ideal S P` as a ring hom -/
+@[simps]
+def coe_ideal_hom : ideal R →+* fractional_ideal S P :=
```

Comment:
Do you think it's worth adding a comment on the `has_coe` instance pointing out the existance of this lemma?